File: Loesungen/loesung_06.py
from library import *


if __name__ == "__main__":
    opt, args = getOpt()
    limite=int(args[0])
    for i in range(10, limite+1):    
        res = sum_proper_divisors(i)
        #print("i %d, Res %d" %(i, res)) 
        if (res - i > 0):
            print("%d ; %d" % (i, res-i))


        

File: Loesungen/loesung_06.cpp
// Bestimmt das kleinste gemeinsame Vielfachte (kgv)
// basic file operations
#include <iostream>
#include "library.h"
//#include "version.h"
using namespace std;


void print_abundant(int limit){
	for (int number=10;number<=limit;number++)
	{
		auto sum = sum_proper_divisors(number);
		if (sum>number){
			std::cout <<number << ", Abundance " << sum-number << std::endl;
		}
	}
	return;
}
	
	
int main (int argc, char *argv[]) {
	int limite;

   std::cout << "Abzundant number" << std::endl;
   std::cout << "Grenze: ";
   std::cin >> limite;
   print_abundant(limite);
   return 0;
}